api/src/main/java/org/apache/brooklyn/api/catalog/CatalogItem.java

```diff
@@ -123,6 +123,8 @@ public String toString() {
     @Nullable public String getIconUrl();
 
     public String getSymbolicName();
+    
+    public String getContainingBundle();
```

Comment:
I've done this and a commit, but then reverted it.  I'm not sure everything comes from a bundle just yet.  When it does we can reinstate.  To me it is tied up with use of search paths etc.